dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -117,6 +117,15 @@ void AnimatedVisualPlayer::AnimationPlay::Start()
             // Subscribe to the batch completed event.
             m_batchCompletedToken = m_batch.Completed([this](winrt::IInspectable const&, winrt::CompositionBatchCompletedEventArgs const&)
                 {
+                    if (m_owner) 
+                    {                        
+                        // If optimization is set to Resources - destroy animations immediately after player stops.
+                        if (m_owner->AnimatedVisualPlayerProperties::AnimationOptimization() == winrt::PlayerAnimationOptimization::Resources)
```

Comment:
>AnimatedVisualPlayerProperties::AnimationOptimization [](http://example.com/codeflow?start=37&length=53)

Being explicit in this way is non-intuitive. The other cases where this ambiguity has come up we've put this in the class in the header file:

using AnimatedVisualPlayerProperties::AnimationOptimization;